src/natcap/invest/ndr/ndr.py

```diff
@@ -1305,9 +1314,9 @@ def _calculate_export_op(
         # these intermediate outputs should always have defined nodata
         # values assigned by pygeoprocessing
         valid_mask = ~(numpy.isclose(modified_load_array, load_nodata) |
-            numpy.isclose(ndr_array, ndr_nodata) |
-            numpy.isclose(modified_sub_load_array, subsurface_load_nodata) |
-            numpy.isclose(sub_ndr_array, sub_ndr_nodata))
+                       numpy.isclose(ndr_array, ndr_nodata) |
+                       numpy.isclose(modified_sub_load_array, subsurface_load_nodata) |
+                       numpy.isclose(sub_ndr_array, sub_ndr_nodata))
```

Comment:
I think line 1318 busts the 80 col mark.

Reply:
I highlighted all those lines to insert a suggestion, but apparently I can't here... Maybe because I selected a non changed line...